lib/main.js

```diff
@@ -1,3 +1,4 @@
+// This is a test.
 var {Hotkey} = require("sdk/hotkeys");
```

Comment:
There is a JavaScript syntax error in your code; validation cannot continue on this file.